Sources/Classes/Public/RSConfigBuilder.h

```diff
@@ -29,6 +29,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype) withConfigRefreshInteval: (int) configRefreshInterval;
 - (instancetype) withTrackLifecycleEvens: (BOOL) trackLifecycleEvents;
 - (instancetype) withRecordScreenViews: (BOOL) recordScreenViews;
+- (instancetype) withBackGroundRunTime:(BOOL)enableBackGroundRunTime;
```

Comment:
Rename the function name to 
`- (instancetype) withEnableBackGroundMode:(BOOL) enableBackgroundMode;`